client/landing/stepper/declarative-flow/internals/steps-repository/store-address/index.tsx

```diff
@@ -249,14 +255,29 @@ const StoreAddress: Step = function StoreAddress( { navigation } ) {
 					</FormFieldset>
 
 					<FormFieldset>
-						<FormLabel htmlFor="store_email">{ __( 'Email address' ) }</FormLabel>
+						<FormLabel htmlFor="store_email">
+							{ __( 'Get Commerce tips straight to your inbox (optional)' ) }
```

Comment:
Should this be "Get WooCommerce tips..."? or "Get commerce tips..."? The capital C is confusing me.

Reply:
That was the copy that [came up in the issue](https://github.com/Automattic/wp-calypso/issues/64202#issuecomment-1163245458).

But "WooCommerce" does make more sense.